Topics/Development_Process/Azure_Functions.md

```diff
@@ -0,0 +1,193 @@
+![image](https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/56797486/efa9cc14-56a4-4d6e-904f-84cc82205a9d)
+
+This will deploy a brand new storage account for Azure. You should then see a "Go to Resource" button. Click it to be taken to the Overview page of your new storage account.
+
+There is a section for "Access Keys" on the left hand side of the Resource Overview page, underneath "Security + Networking". Click it, and reveal the key and connection string for either key. Put these values in a Notepad.
+![image](https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/56797486/c5d15172-e610-4ee7-9624-d693ae01fcde)
+
+
+### Create Azure Function App
+
+Go back to the Azure Portal Home and search "Function App" in the Navigation. Select the resulting "Function App" and click "Create Function App".
+
+Inside the Create Function App menu, we will configure the following:
+
+- Subscription and Resource Group the same as above
+- Function app name will be `yournamehere-func-demo`
+- Runtime stack will be .NET 6 (choose your own if desired)
+- Region is US East
+- OS will be Windows
+- Hosting will be Consumption
+
+Select "Review + Create" and ensure your configuration for the Azure Function App is correct. Then select "Create".
+![image](https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/56797486/61488aad-ef1a-41ce-8ba2-eac54f4de3bf)
+
+## System Setup
+
+First, install the [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli). I am on MacOS using Homebrew, so I will open up a terminal and run the following:
+
+```bash
+brew update && brew install azure-cli
+```
+
+Since we will be developing our functions in our local machine, go ahead and install the [Azure Functions Core Tools](https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local). Again, I will install using Homebrew:
+
+```bash
+brew tap azure/functions
+brew install azure-functions-core-tools@4
+```
+
+Finally, this tutorial will be using C# as language of choice. Install the dotnet framework:
+
+```bash
+brew install dotnet@6
+```
+
+and add to your path:
+
+```bash
+echo 'export PATH="/opt/homebrew/opt/dotnet@6/bin:$PATH"' >> ~/.zshrc
+```
+
+Azure Functions can be developed in other languages like Python and JavaScript, so feel free to choose a different language if you'd like.
+
+_Optional_: If you are using VS Code, go ahead and install the Azure Functions extension as well. This extension (and other Azure extensions) work really well with VS Code.
+
+## Create an Azure Function
+
+Open up a terminal and `cd` into a directory you'd like to create our Azure Function project. Once there, run `func init`. Select `dotnet` followed by `c#`, and your Azure Functions project will be initialized.
```

Comment:
Small typo: Should be 

"into a directory you'd like to create _your_ Azure Function project"

Reply:
Good catch, fixed!